dist/s_whitespace

```diff
@@ -24,6 +24,8 @@ find bench dist examples ext src test \
     -name '*.dox' -o \
     -name '*.in' -o \
     -name 's_*' -o \
+    -name '*.sh' -o \
```

Comment:
One np, can you have this list alphabetized, please?

Reply:
ah, sorry, I forgot I left this open. Done.

Reply:
Thanks!